jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdContainer.java

```diff
@@ -63,6 +64,8 @@ public class EtcdContainer extends GenericContainer<EtcdContainer> {
     private Path dataDirectory;
     private Collection<String> additionalArgs;
     private boolean shouldMountDataDirectory = true;
+    private boolean rootless = true;
```

Comment:
I don't think this is needed, you may require to switch the user regardless of the fact that docker is in ritlees mode or not. Also, on other container tools such as podman it does not make sense

Reply:
- Removed in new commmit.